cmd/kubeadm/app/constants/constants.go

```diff
@@ -431,6 +431,15 @@ const (
 	EtcdUserName string = "kubeadm-etcd"
 	// ServiceAccountKeyReadersGroupName is the group of users that are allowed to read the service account private key.
 	ServiceAccountKeyReadersGroupName string = "kubeadm-sa-key-readers"
+
+	// CoreDNSPhase is the name of CoreDNS phase in "kubeadm init"
+	CoreDNSPhase string = "addon/coredns"
+
+	// KubeProxyPhase is the name of kube-proxy phase during "kubeadm init"
+	KubeProxyPhase string = "addon/kube-proxy"
+
+	// AddonPhase is the name of addon phase during "kubeadm init"
+	AddonPhase string = "addon"
```

Comment:
please move these to init.go where they are used above the manageSkipAddons function, lowercase them as well (make them local)

Reply:
Done.